libs/blocks/merch-card/merch-card.js

```diff
@@ -249,6 +249,12 @@ const parseContent = async (el, merchCard) => {
     if (mnemonicList) bodySlot.append(mnemonicList);
   });
 
+  if (calloutContainer.children.length > 0) {
+    const calloutSlot = createTag('div', { slot: 'callout-content' });
```

Comment:
Why not flatten the DOM and just add slot attribute to `calloutContainer`? 
```suggestion
    callloutContainer.setAttribute('slot', 'callout-content');
```

Reply:
If it's like that already in mas code, let's leave it as is.